tests/test_functional.py

```diff
@@ -276,7 +276,7 @@ def test_obal_release_downstream_hello_wait_download_rpms():
 def test_obal_update_downstream_hello():
     setup_upstream('../upstream/')
 
-    assert_obal_success(['update', 'hello'])
+    assert_obal_success(['update', 'hello', '-e', 'version=2.10', '-e', 'downstream=true'])
```

Comment:
why is `downstream=true` needed? I can't find such a variable anywhere in our codebase.

also, I don't think an downstream update should require a version, as that scenario should be "just copy over stuff", no?

Reply:
oooh, but you use the version in the commit msg, fair enough.

Reply:
passes localy w/o `downstream=true`